types/requestidlecallback/index.d.ts

```diff
@@ -19,3 +19,8 @@ export interface IdleDeadline {
 export interface IdleRequestOptions {
 	timeout: number;
 }
+
+export interface Window {
+  requestIdleCallback(callback: IdleRequestCallback): number;
```

Comment:
missing options arg.

Reply:
I think you should just convert previous function declarations to methods and use `IdleCallbackHandle` instead of `number`.